frontend_gen.py

```diff
@@ -5,23 +5,23 @@
 
 # Page String Components
 page_header = """
-import React from 'react';
-import { Responsive, WidthProvider } from 'react-grid-layout';
-import TableView from '../Table/TableView';
-import ReactMarkdown from 'react-markdown';
-import MenuBar from '../MenuBar/MenuBar';
+import React, { Suspense } from 'react'
+import { Responsive, WidthProvider } from 'react-grid-layout'
+import MenuBar from '../MenuBar/MenuBar'
+import { Spin } from 'antd'
 import './Page.css'
-import FullPlotly from '../Plots/FullPlotly'
-import Metadata from '../Table/Metadata'
-import DynamicGrid from '../DynamicGrid'
-import Image from '../Image'
-import Markdown from '../Markdown'
```

Comment:
Did you remove some of these to optimize the loading? To make it lazier?

Reply:
Correct, the imports are now only added if its corresponding component is present on the page. Additionally the components TS file is only loaded when it is rendered and not when the user first lands on the site. These two optimizations should improve the UX with regards to load time of pages and minimize the deployment memory footprint.